src/bcccp/tickets/season/SeasonTicket.java

```diff
@@ -3,6 +3,7 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Iterator;
 
 public class SeasonTicket implements ISeasonTicket {
```

Comment:
1. TYPE. Google Style Infringement
No javadocs present:
"At the minimum, Javadoc is present for every public class,
 and every public or protected member of such a class".